metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchDoc.java

```diff
@@ -94,6 +120,10 @@ public static ElasticSearchDoc parse(final GetResponse response) {
 
     private ObjectNode toJsonObject() {
         final ObjectNode oMetadata = MetacatJsonLocator.INSTANCE.toJsonObject(dto);
+        if (addSearchableDefinitionMetadata) {
```

Comment:
it's kind of confusing here to have the variable and the function have exactly the same name